adcm_aio_client/core/objects/_imports.py

```diff
@@ -1 +1,47 @@
-class ClusterImports: ...
+from typing import TYPE_CHECKING, Collection, Iterable, Self, Union
+
+from adcm_aio_client.core.types import Endpoint, Requester
+
+if TYPE_CHECKING:
+    from adcm_aio_client.core.objects.cm import Cluster, Service
+
+
+class Imports:
+    def __init__(self: Self, requester: Requester, path: Endpoint) -> None:
+        self._requester = requester
+        self._path = path
+
+    async def _get_source_binds(self: Self) -> set[tuple[int, str]]:
+        response = await self._requester.get(*self._path)
+        data_binds = set()
+
+        for import_data in response.as_dict()["results"]:
+            binds = import_data.get("binds", [])
+            for bind in binds:
+                bind_id = int(bind["source"].get("id"))
```

Comment:
`get` may return `None` `int(None)` will fail

IMO better to fail with `bind["source"]["id"]` (on absence of `id` than on something like "None is not convertible to int")